tcl/interface/ftdi/minimodule.cfg

```diff
@@ -4,14 +4,13 @@
 # http://www.ftdichip.com/Support/Documents/DataSheets/Modules/DS_FT2232H_Mini_Module.pdf
 #
 
-echo "WARNING!"
-echo "This file was not tested with real interface, it is based on code in ft2232.c."
-echo "Please report your experience with this file to openocd-devel mailing list,"
-echo "so it could be marked as working or fixed."
-
 interface ftdi
 ftdi_device_desc "FT2232H MiniModule"
 ftdi_vid_pid 0x0403 0x6010
 
-ftdi_layout_init 0x0018 0x05fb
-ftdi_layout_signal nSRST -data 0x0020
+# Every pin set as high impedance except TCK, TDI, TDO and TMS
```

Comment:
hmm, I was using this one. Not sure why it needed to change. I won't be able to test it for a week or so though, I guess it shouldn't block this PR but just a note that it changed.

Reply:
```
commit 5a6f0495db71ef1c05a912cd87e52e3d62e641a3
Author: Diego Herranz <diegoherranz@diegoherranz.com>
Date:   Thu Aug 3 07:37:41 2017 +0100

    tcl/interface/ftdi: improve minimodule config
    
    - Tested on a real FT2232H MiniModule, so warning removed.
    - Every pin initially set to high impedance except TCK, TDI,
      TDO and TMS: Safest values given it's an evaluation board
      and the rest of pins might be connected to something else.
    - Reset is now initially de-asserted (it was asserted
      which is not recommended).
    - nRST pin choice is arbitrary so comment added (wondering
      if it should be an "echo").
    - "-oe" option added to NRST signal so it can be set as
      high impedance (tri-stated).
    
    Change-Id: I967ab0c7bbccf72dbf6d6d78b3180b74e016e0d6
    Signed-off-by: Diego Herranz <diegoherranz@diegoherranz.com>
    Reviewed-on: http://openocd.zylin.com/4185
    Tested-by: jenkins
    Reviewed-by: Spencer Oliver <spen@spen-soft.co.uk>

diff --git a/tcl/interface/ftdi/minimodule.cfg b/tcl/interface/ftdi/minimodule.cfg
index 57249dfc..7df096d9 100644
--- a/tcl/interface/ftdi/minimodule.cfg
+++ b/tcl/interface/ftdi/minimodule.cfg
@@ -4,14 +4,13 @@
 # http://www.ftdichip.com/Support/Documents/DataSheets/Modules/DS_FT2232H_Mini_Module.pdf
 #
 
-echo "WARNING!"
-echo "This file was not tested with real interface, it is based on code in ft2232.c."
-echo "Please report your experience with this file to openocd-devel mailing list,"
-echo "so it could be marked as working or fixed."
-
 interface ftdi
 ftdi_device_desc "FT2232H MiniModule"
 ftdi_vid_pid 0x0403 0x6010
 
-ftdi_layout_init 0x0018 0x05fb
-ftdi_layout_signal nSRST -data 0x0020
+# Every pin set as high impedance except TCK, TDI, TDO and TMS
+ftdi_layout_init 0x0008 0x000b
+
+# nSRST defined on pin CN2-13 of the MiniModule (pin ADBUS5 [AD5] on the FT2232H chip)
+# This choice is arbitrary. Use other GPIO pin if desired.
+ftdi_layout_signal nSRST -data 0x0020 -oe 0x0020
```